test/release/examples/benchmarks/lcals/README

```diff
@@ -30,35 +30,31 @@ This directory's contents are as follows:
   RunSPMDRawLoops.chpl          : /
   RunVectorizeOnlyRawLoops.chpl :/
 
-  LCALSConfiguration.chpl : Configuration constants to define what to run
-  LCALSLoops.chpl         : Routines to initialize kernel data
+  LCALSConfiguration.chpl   : Configuration constants to define what to run
+  LCALSLoops.chpl           : Routines to initialize kernel data
 
-  LCALSDataTypes.chpl : | Data types and enumerations
-  LCALSEnums.chpl     :/
+  LCALSDataTypes.chpl   :\ Data types and enumerations
+  LCALSEnums.chpl       :/
 
-  LCALSParams.chpl : | Global variables and constants
-  LCALSStatic.chpl :/
+  LCALSParams.chpl      :\ Global variables and constants
+  LCALSStatic.chpl      :/
 
-  LCALSChecksums.chpl : Checks the validity of computed checksums
+  LCALSChecksums.chpl   : Checks the validity of computed checksums
 
-  Timer.chpl      : A simple timer
+  Timer.chpl            : A simple timer
 
-  LongDouble.chpl : | An interface to the "long double" type in C
-  longdouble.h    :/
+  LongDouble.chpl       :\  An interface to the "long double" type in C
+  longdouble.h          :/
 
-  Makefile       : Build LCALS
+  Makefile              : Build LCALS
 
-  *.notest               :\
-  LCALS-*.graph          : \
-  LCALSMain.perfcompopts :  \
-  LCALSMain.perfexecopts :   \ Used by the Chapel testing system
-  LCALSMain.perfkeys     :   /
-  LCALSMain.perftimeout  :  /
-  LCALSMain.skipif       : /
-  LCALSMain.timeout      :/
+  README                    : This file
+  README-LCALS_license.txt  : The LCALS reference license
 
-  README                   : This file
-  README-LCALS_license.txt : The LCALS reference license
+  Other files with the same base filenames as the Chapel test programs but
+  different filename extensions, are for use by the automated test system.
+  For example, file Timer.notest supports automated testing of Timer.chpl.
```

Comment:
I think you should pick a different example here.  Timer.notest supports _**not**_ testing Timer.chpl.  A better example would be LCALSMain.skipif and LCALSMain.chpl.